scripts/blocklist-skylink.sh

```diff
@@ -37,45 +34,18 @@ else
     skylinks=("$1") # just single skylink passed as input argument
 fi
 
-# Block skylinks in batches
-skylinks_len=${#skylinks[@]}
-for (( i = 0; i < $skylinks_len; i++ )); do
-    # Add skylink to batch
-    skylink="${skylinks[$i]}"
-    echo ".. ⌁ Adding skylink ${skylink} to batch..."
-    batch_skylinks+=("$skylink")
-
-    # For performance reasons on each iteration we do not block a single
-    # skylink, but we block skylinks in batches with BATCH_SIZE size mainly
-    # because of nginx cache search.
-    # If (batch len == batch size) or (we have last batch):
-    if (( ${#batch_skylinks[@]} == $BATCH_SIZE || $i == $skylinks_len - 1 )); then
-        echo "--------------------------------------------"
-
-        # Add to Sia blocklist
-        echo "Blocking batch skylinks in skyd..."
-        skylinks_space_separated="$(IFS=' '; echo "${batch_skylinks[*]}")"
-        docker exec sia siac skynet blocklist add $skylinks_space_separated
+# get local nginx ip adress
+nginx_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nginx)
```

Comment:
nit: A quick check that we managed to get an IP (i.e. nginx is running) might be nice. Not blocking, though.

Reply:
If nginx wouldn't be running we would get `Error: No such object: nginx` from the docker command